### PR TITLE
Background check bugfix

### PR DESCRIPTION
**Pull request by adrichey**
Addresses #1740.

### CODE REVIEW

**Comment by adrichey** [comment]
This is ready for review.

File: app/javascript/mentor/store/getters.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default {
   },
 
   isBackgroundCheckWaived (state) {
-    const isCountryUS = digStateAttributes(state, 'currentAccount', 'countryCode', code => code == 'US')
```
Comment (joemsak) [inline app/javascript/mentor/store/getters.js]: countryCode .... should be available (and more reliable for formatting comparison purposes. is it not?

**Comment by joemsak** [comment]
I'm still not understanding what I'm seeing in this PR, I'm sorry. We might have to get on a call

**Comment by adrichey** [comment]
Closing until we have more of a chance to talk about future workflows and have time to start addressing our wishlist items.

**Pull request closed without merging**